src/DataTypes/Serializations/SerializationNullable.cpp

```diff
@@ -626,9 +626,23 @@ template<typename ReturnType>
 ReturnType SerializationNullable::deserializeTextJSONImpl(IColumn & column, ReadBuffer & istr, const FormatSettings & settings,
                                                     const SerializationPtr & nested)
 {
-    return safeDeserialize<ReturnType>(column, *nested,
-        [&istr] { return checkStringByFirstCharacterAndAssertTheRest("null", istr); },
-        [&nested, &istr, &settings] (IColumn & nested_column) { nested->deserializeTextJSON(nested_column, istr, settings); });
+    return safeDeserialize<ReturnType>(
+        column,
+        *nested,
+        [&istr, &column] {
+            if (column.isNullable())
+            {
+                /// If the column is nullable and the value field is empty, we assume it is NULL.
+                /// eg. {"key": ""}
+                if (*istr.position() == '"' && (*(istr.position() + 1) == '"'))
```

Comment:
there is a special situation, if the nested_column is string such as `nullable(string)` and the filed is empty , it will output null instead empty string, should i add a judgment to deal with it?

Reply:
yes, for `string` and `fixed_string` should be handled specially, (only supports explicit "null")